modules/monitoring/manifests/checks/sidekiq.pp

```diff
@@ -38,7 +38,7 @@
     # many false positives from not much activity, while also ensuring
     # that if there is no data for longer than 24 hours, the alert
```

Comment:
Should this also be "5 hours"?

Reply:
Yep, good spot. I've updated the comment now.